scanning/disa-alignment/ansible/main.fmf

```diff
@@ -0,0 +1,33 @@
+summary: Compare SSG and DISA STIG benchmark scan results after Ansible remediation
+test: python3 -m lib.runtest ./test.py
+result: custom
+environment+:
+    PYTHONPATH: ../../..
+require+:
+  # virt library dependencies
+  - libvirt-daemon
+  - libvirt-daemon-driver-qemu
+  - libvirt-daemon-driver-storage-core
+  - libvirt-daemon-driver-network
+  - firewalld
+  - qemu-kvm
+  - libvirt-client
+  - virt-install
+  - rpm-build
+  - createrepo
+  - ansible-core
+  - rhc-worker-playbook
```

Comment:
These two are not "virt library dependencies", I would probably copy/paste them along with their comments from `hardening/ansible/main.fmf`.